common/static/coffee/src/discussion/views/discussion_thread_view.coffee

```diff
@@ -19,6 +19,7 @@ if Backbone?
     initialize: (options) ->
       super()
       @mode = options.mode or "inline"  # allowed values are "tab" or "inline"
+      @context = options.context or "course"  # allowed values are "course" or "standalone"
```

Comment:
What is the relationship between discussion_module_view.coffee and this file (why does the context need to be passed through in both places)?